package.json

```diff
@@ -53,8 +53,8 @@
     "style:fix": "yarn fmt:fix && yarn lint:fix",
     "fmt": "prettier -c ./**/*.{js,ts}",
     "fmt:fix": "prettier -w ./**/*.{js,ts}",
-    "lint": "eslint --ext .js,.ts,.tsx .",
```

Comment:
`--ext` is deprecated and removed, we just include the proper files now with globs pattern.